src/main/java/bob/BobTaskList.java

```diff
@@ -0,0 +1,218 @@
+package bob;
+
+import java.util.ArrayList;
+
+/**
+ * Class that handles the task list of the chat bot.
+ */
+public class BobTaskList {
+
+    private ArrayList<Task> list;
+    private BobStorage storage;
+    private BobUI ui;
+
+    /**
+     * Constructor of BobTaskList.
+     *
+     * @param storage Chatbot's storage component.
+     * @param ui Chatbot's UI component.
+     */
+    public BobTaskList(BobStorage storage, BobUI ui) {
+        this.storage = storage;
+        this.ui = ui;
+    }
+
+    public void setList(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    public ArrayList<Task> getList() {
+        return this.list;
```

Comment:
Referring to the comment regarding the unnecessary of of `this`, you may omit `this.` here.